src/widgets/config/RegistrationPageConfig.ts

```diff
@@ -44,11 +44,23 @@ export default class RegistrationPageConfig extends WidgetConfig {
      */
     readonly numberOfTickets: boolean;
 
+    /**
+     * Array of country codes to use for the registration form
+     */
+    readonly countryCodes: Array<string>;
+
+    /**
+     * Preselected default country
+     */
+    readonly countryDefault: string;
+
```

Comment:
@cluk33 I like the idea yet the configuration can be improved IMHO. I suggest uniting new parameters under a new parent as they are about one element.  For example, 

```
countries: {
   codes: [],
   default: 'DE'
}
```

I also think the name `countries.codes` does not say anything about the usage. If it's undefined, then all countries are shown, not selected one, making the comment to the parameter incorrect. A name like 'countries.allowed' or 'countries.onlyFrom' is better.